docs/attributes-registry/url.md

```diff
@@ -8,15 +8,37 @@ linkTitle: URL
 <!-- semconv registry.url(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
+| `url.domain` | string | Domain of the url, such as "opentelemetry.io". [1] | `www.foo.bar`; `opentelemetry.io`; `3.12.167.2`; `[1080:0:0:0:8:800:200C:417A]` |
+| `url.extension` | string | The file extension from the original request url, excluding the leading dot. [2] | `png`; `gz` |
 | `url.fragment` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>The [URI fragment](https://www.rfc-editor.org/rfc/rfc3986#section-3.5) component | `SemConv` |
-| `url.full` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Absolute URL describing a network resource according to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) [1] | `https://www.foo.bar/search?q=OpenTelemetry#SemConv`; `//localhost` |
+| `url.full` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Absolute URL describing a network resource according to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) [3] | `https://www.foo.bar/search?q=OpenTelemetry#SemConv`; `//localhost` |
+| `url.original` | string | Unmodified original URL as seen in the event source. [4] | `https://www.foo.bar/search?q=OpenTelemetry#SemConv`; `search?q=OpenTelemetry` |
+| `url.password` | string | Password of the request. | `password` |
 | `url.path` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>The [URI path](https://www.rfc-editor.org/rfc/rfc3986#section-3.3) component | `/search` |
-| `url.query` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>The [URI query](https://www.rfc-editor.org/rfc/rfc3986#section-3.4) component [2] | `q=OpenTelemetry` |
+| `url.port` | int | Port of the request | `443` |
```

Comment:
How do these relate to [server.port](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/server.md) and [client.port](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/client.md)?

Reply:
If we want to avoid semantic meaning here, I think we should probably change the descriptions of things like `url.port` from "Port of the request" to "Port described in the URL".

I.e. these attributes can exist to fully describe components of a URL.  I think in our instrumentation recomendations we'll likely prefer using semantics like `server` or `client`, but I also think it's fine to be able to fully describe a URL here.

Reply:
Thanks @jsuereth. I think correct rephrasing that these attributes are extracted from the url itself should resolve the issue.